meta-mender-tegra/recipes-mender/tegra-state-scripts/files/redundant-boot-install-script-uboot

```diff
@@ -26,22 +26,27 @@ if [ $num_slots != 2 ]; then
         exit 1
     fi
 fi
-current_slot=`nvbootctrl get-current-slot`
-echo "Current boot slot: $current_slot"
 
-new_boot_part=`fw_printenv -n mender_boot_part`
```

Comment:
I'm concerned setting `new_boot_part` which doesn't match mender settings may cause other issues.  I'm guessing you are in a situation where your boot/root slot alignment isn't proper due to issues like the ones discussed in https://github.com/BoulderAI/meta-mender-community/pull/1#issue-516955713 ?  It might be beter to try to detect/correct that problem instead of using something other than `mender_boot_part` to select the new boot partition.

Reply:
yes, sure. I'll check that out